internal/rest/dequeue_nodes.go

```diff
@@ -108,6 +110,16 @@ func (rest *RestOperations) DequeueNodes(key string) {
 			rest.vrfCU(key, name, avimodel)
 			return
 		}
+		if strings.Contains(name, "StringGroup") {
```

Comment:
`string group` and `datscript` functionality is related to EVH model only. So it should be handled in `RestOperationEVH`. We should have `name` enqueued as per that only. What is the reason of including this functionality here.